docs/integration_resources/resources/usd.md

```diff
@@ -0,0 +1,29 @@
+# USD
+
+Shift has developed a specialized catalog and utility plugins for using the USD API in Shift. 
+
+* [Shift_USD](https://github.com/Inbibo/Shift_USD)
+
+## Installation
+
+To start working with the USD API in Shift, please follow the following steps:
+
+* Download the necessary dependencies (Python & USD).
+* Clone the [Shift_USD](https://github.com/Inbibo/Shift_USD) repository into the preferred folder.
+* Add the path to the Shift_USD directory to the `PYTHONPATH` environment variable.
+* Add the path to the Shift_USD directory to the `SHIFT_CATALOG_PATH` environment variable.
+* Add the path of the *plugin.json* file found in the Shift_USD directory to the `SHIFT_PLUGIN_PATH` environment variable.
+
+## Dependencies
+| **Dependency**                                           | **Version** |
+| -------------------------------------------------------- | ----------- |
+| [Python](https://www.python.org/download/releases/3.0/)  | > 3.7       |
+| [USD](https://graphics.pixar.com/usd/release/index.html) | > 21.8      |

```

Comment:
Why do we need USD version to be higher than 21.8?
Also maybe we should add the link to the usd-core PyPi page as that is the minimal requirement for the catalog to work.
This page points to code that needs to be compiled so it is not ready to use.

Reply:
I changed the wording, although I would like confirmation from @hectorjimenezinbibo and you that it is true or if there's a version the catalog is not compatible anymore.

Reply:
Well, I think is correct showing this information, because taking a look to the `usd-core` [releases](https://pypi.org/project/usd-core/#history) page: `usd-core` 21.5 is compatible with Python >= 3.6 and Python < 3.8, but `usd-core` 21.8 is compatible with Python > 3.6 and Python < 3.10, which means that if we're planning to use as base Python version 3.7 or Python 3.10, the basic one that will be compatible with both Python versions is 21.8.